src/user/models.py

```diff
@@ -14,16 +21,15 @@ class RoleEnum(Enum):
 
 class User(Base):
     __tablename__ = "users"
-
     id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
     name: Mapped[str] = mapped_column(Text)
     surname: Mapped[str] = mapped_column(Text)
     username: Mapped[str] = mapped_column(Text, unique=True)
     phone_number: Mapped[str] = mapped_column(Text, unique=True)
     email: Mapped[str] = mapped_column(Text, unique=True)
-    role: Mapped[RoleEnum]
+    role: Mapped[RoleEnum] = mapped_column(default=RoleEnum.USER)
     group_id: Mapped[int] = mapped_column(Integer, ForeignKey("groups.id"))
-    created_at: Mapped[datetime]
+    created_at: Mapped[datetime.datetime]
```

Comment:
use Mapped[timestamp] which you declared above (lines 10-15), it will has datetime.datetime type, but useful constrains as well